### ISSUE TITLE
New whole verse bookmark icons proposal

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#1189]
My preference for indicating whether the icon is active is to use an alternative icon rather than a modified color. Modified color is good usually but we have so many colors that it can be a little hard to follow.

I have some SVGs.
![Verse selected icons](https://user-images.githubusercontent.com/13920678/123419930-2e4ade00-d5fe-11eb-8184-1361066f625b.png)

![edit icon in context](https://user-images.githubusercontent.com/13920678/123419795-03608a00-d5fe-11eb-9be8-28f23294d570.png)

**Comment by tuomas2** [comment]
What about having other icon with arrows, other without? V could be something else also (V is verse in english, but it's not very international). 

But good idea, I'll go with this if no better option arises.

**Comment by agrogers** [comment]
I've been thinking about alternatives to the letter V but haven't come up with anything better. Best idea is just a series of horizontal lines with arrow underneath. But perhaps that won't be clear to anyone then, even English speakers 😏

**Comment by bvahedy** [comment]
Although I like the 529.1 version, but How about something like this:

**Comment by agrogers** [comment]
'this' seems to be missing Ben

**Comment by bvahedy** [comment]
Sorry, I forgot Gmail doesn't bring images over. Of course this is a rough draft:
![sketch1624778241938](https://user-images.githubusercontent.com/46405809/123536916-d4bbdf80-d6e1-11eb-9a4e-45421e9bb7db.png)

**Comment by tuomas2** [comment]
Let's handle this also on #1220 and close here
